### PR TITLE
Ssh config

### PR DESCRIPTION
**Pull request by sohlcoveo**
Add config script for ssh and bitbucket.

We add: 
- openssh
- config ssh for bitbucket
- public key

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
 Built from the official hashicorp/terraform:latest container.
 
 We try to keep it up-to-date, if you want latest version you can do a pull request.
+
+## Config:
+
+Add environment variable :
+
+```
+BITBUCKET_PUB_KEY     = Public ssh key
```
Comment (pastjean) [inline README.md]: `BITBUCKET_PUB_KEY` i wonder why is is named like that instead of `SSH_PUBLIC_KEY`

File: entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+mkdir /root/.ssh
+
+# Add Public key
+echo $BITBUCK_PUB_KEY > /root/.ssh/id_rsa
```
Comment (pastjean) [inline entrypoint.sh]: `BITBUCK_PUB_KEY` is named `BITBUCKET_PUB_KEY` higher up but as i said `SSH_PUBLIC_KEY` PLZ

File: README.md (hunk 1 above)
Comment (sohlcoveo) [inline README.md]: Because it is just for bitbucket, the ssh config is setted for bitbucket. In the future we can add other ssh key.

Comment (pastjean) [inline README.md]: BITBUCKET_SSH_PUBLIC_KEY then for clarity ?

**Pull request merged**